examples/miscellanea/cartopy_gridspec.py

```diff
@@ -0,0 +1,100 @@
+"""
+Using Cartopy and AxesGrid toolkit
+----------------------------------
+This example demonstrates how to use cartopy `GeoAxes` with
+matplotlib's gridspec. Creates a list of axes for a figure
+with gridspec layout of pre-defined number of rows and columns
+
+Parameters:
+rows (int): Number of rows in grid
+cols (int): Number of columns in grid
+proj (cartopy.crs projection): cartopy.crs projection to be used
+
+Returns:
+axlist: List of axes for each individual figure in the grid.
+
+>>> make_grid(2,2,ccrs.PlateCarree())
+[<AxesSubplot:>, <AxesSubplot:>, <AxesSubplot:>, <AxesSubplot:>, <AxesSubplot:>]
+
+>>> make_grid(2,2,ccrs.PlateCarree())
+[<AxesSubplot:>, <AxesSubplot:>, <AxesSubplot:>, <AxesSubplot:>]
```

Comment:
This is all extraneous and incorrect. The first iteration you have 6, then 4 but with the same call? I'd suggest removing this entire chunk starting from "Creates a list of axes" and then expanding more upon why you want to use gridspec here and what you need to do  special (only pass in a projection parameter?).